app/views/rentals/_form.html.erb

```diff
@@ -1,85 +1,87 @@
+        </div>
+      </div>
 
-  <div class="form-group col-xs-12">
-    <%= f.label :start_time, 'Pickup Date', class: 'control-label col-xs-2' %>
-    <div class="col-xs-6">
-      <%= f.date_field :start_time, value: @start_date, required: true, id: 'rental_start_time', class: "form-control cost-dependent" %>
-    </div>
-  </div>
+      <div class="form-group row">
+        <%= f.label :start_time, 'Pickup Date', class: 'control-label col-xs-3' %>
+        <div class="col-xs-9">
+          <%= f.date_field :start_time, value: @start_date, required: true, id: 'rental_start_time', class: "form-control cost-dependent" %>
+        </div>
+      </div>
 
-  <div class="form-group col-xs-12">
-    <%= f.label :end_time, 'Dropoff Date', class: 'control-label col-xs-2' %>
-    <div class="col-xs-6">
-      <%= f.date_field :end_time, value: @start_date, required: true, id: 'rental_end_time', class: "form-control cost-dependent" %>
-    </div>
-  </div>
+      <div class="form-group row">
+        <%= f.label :end_time, 'Dropoff Date', class: 'control-label col-xs-3' %>
+        <div class="col-xs-9">
+          <%= f.date_field :end_time, value: @start_date, required: true, id: 'rental_end_time', class: "form-control cost-dependent" %>
+        </div>
+      </div>
 
 
-  <div class="form-group col-xs-12">
-    <%= label_tag :amount, 'Cost', class: 'control-label col-xs-2' %>
-    <div class="col-xs-6">
-      <%= number_field_tag :amount, nil, min: 0, required: true, disabled: !@current_user.has_permission?("rentals", "cost_adjustment"), id: 'rental_amount', class: 'form-control' %>
+      <div class="form-group row">
+        <%= label_tag :amount, 'Cost', class: 'control-label col-xs-3' %>
+        <div class="col-xs-9">
+          <%= number_field_tag :amount, nil, min: 0, required: true, disabled: !@current_user.has_permission?("rentals", "cost_adjustment"), id: 'rental_amount', class: 'form-control' %>
+        </div>
+      </div>
+
+      <div class="form-group row">
+        <%= label_tag :date_range, 'Number of Days', class: 'control-label col-xs-3' %>
+        <div class="col-xs-9">
+          <%  #(:start_time - :end_time) %>
+          <%= number_field_tag :date_range, 1, min: 0, disabled: true, class: 'form-control' %>
+        </div>
+      </div>
     </div>
-  </div>
 
-  <div class="form-group col-xs-12">
-    <%= label_tag :date_range, 'Number of Days', class: 'control-label col-xs-2' %>
     <div class="col-xs-6">
-      <%  #(:start_time - :end_time) %>
-      <%= number_field_tag :date_range, 1, min: 0, disabled: true, class: 'form-control' %>
+      <div class="list-group">
+        <% @item_types.each do |it| %>
+          <div class="input-group selectable-item-type">
+            <%= link_to it.name, '#', class: 'list-group-item' %>
+            <!-- we use _name attr because any ele with a name attr is submitted to server, this is a dummy field, we have to fix it latter so it submitts properly -->
```

Comment:
You're aware that HTML comments will be rendered in your DOM and inspectable by your users?

Reply:
yes, they can also find this project open source on github

Reply:
True. It's also just additional bulk in your HTML that's really just meant internally for your developers. It's not a huge security concern, but doesn't seem like what you want. Particularly there are correctly done ERB comments on this very page.